server/core/src/fulltext.ts

```diff
@@ -165,12 +179,12 @@ export class FullTextIndex implements WithFind {
       return true
     })
 
-    const fullTextLimit = options?.limit ?? 200
+    const fullTextLimit = Math.min(5000, (options?.limit ?? 200) * 100)
```

Comment:
It will not work, elastic has 10000 max limit

Reply:
yep, what's why it min of 5000 and value :-)